docs/source/CONTRIBUTING.md

```diff
@@ -107,10 +107,11 @@ lifting. This includes managing development environments! If you are not
 familiar with this tool, we highly recommend checking out [their docs][poetry docs]
 to get used to the basic usage.
 
-Now, setting up a development environment is super simple!
+Now, setting up a development environment is super simple! Additional info if you run into trouble: [Poetry Environments]
 
 Step 1: [Install Poetry]  
 Step 2: Run ``poetry install``
+Step 3: Optionally Run ``poetry shell``
```

Comment:
Super hard to tell there is anything wrong here. I only noticed when browsing locally, and because I have visible whitespace in my IDE. In Markdown, you have to put two blank spaces at the end of a line, to force a line break. It's a PITA, I know.
```suggestion
Step 2: Run ``poetry install``  
Step 3: Optionally Run ``poetry shell``
```

Reply:
👍  nice catch, I missed that